docs/support-and-maintenance/index.md

```diff
@@ -0,0 +1,96 @@
+# Support and Maintenance
+
```

Comment:
It's a bit weird to have two headers following eachother like this without any text in between. Maybe you could have a short introduction.